pkgs/development/python-modules/aioprometheus/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, orjson
+, quantile-python
+, aiohttp
+, starlette
+, quart
+, pytestCheckHook
+, asynctest
+, httpx
+, fastapi
+, uvicorn
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "aioprometheus";
+  version = "22.5.0";
+  format = "setuptools";
+
+  disable = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "claws";
+    repo = "aioprometheus";
+    rev = version;
+    hash = "sha256-oXLKJDCJFRcR+yOXpTVuQgrnOfPRnS8g51Aci/qMOOw=";
+  };
+
+  propagatedBuildInputs = [
+    orjson
+    quantile-python
+    aiohttp
+    starlette
+    quart
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    asynctest
+    httpx
+    fastapi
+    uvicorn
+  ];
+
+  pytestFlagsArray = [
+    # AttributeError: 'NoneType' object has no attribute 'app'
+    "--deselect=tests/test_fastapi.py::TestFastAPIUsage::test_asgi_middleware"
+
+    # AssertionError: no logs of level ERROR or higher triggered on quart.app
+    "--deselect=tests/test_quart.py::TestQuartRender::test_asgi_middleware"
+    "--deselect=tests/test_quart.py::TestQuartRender::test_asgi_middleware_group_status_codes_enabled"
+  ];
```

Comment:
The deselected tests should handled with `disabledTests`.

Reply:
I haven't found a way to do it with tests that have the same name, as e.g. `test_asgi_middleware` here. What should I do in such cases?

Reply:
We do not need it now as test are no more failing